include/MixerChannelView.h

```diff
@@ -58,32 +56,34 @@ class MixerChannelView : public QWidget
 	Q_PROPERTY(QColor strokeInnerActive READ strokeInnerActive WRITE setStrokeInnerActive)
 	Q_PROPERTY(QColor strokeInnerInactive READ strokeInnerInactive WRITE setStrokeInnerInactive)
 public:
+	static constexpr auto InnerBorderSize = 3;
+	static constexpr auto OuterBorderSize = 1;
+
 	MixerChannelView(QWidget* parent, MixerView* mixerView, int channelIndex);
 	void paintEvent(QPaintEvent* event) override;
 	void contextMenuEvent(QContextMenuEvent*) override;
 	void mousePressEvent(QMouseEvent*) override;
 	void mouseDoubleClickEvent(QMouseEvent*) override;
 	bool eventFilter(QObject* dist, QEvent* event) override;
 
-	int channelIndex() const;
+	void reset() { 	m_peakIndicator->resetPeakToMinusInf(); }
```

Comment:
What's the advantage of moving this method into the header? It just exposes implementation details which can be seen by the additional `PeakIndicator` include which is needed now.

Performance also shouldn't be an issue as we are not resetting mixer channel views in tight loops but it should be called rather seldom.

Reply:
Hey @michaelgregorius, I moved the reset function back into the implementation file, removed the `PeakIndicator.h` include, and added the forward declaration back in.